gulpfile.js

```diff
@@ -51,6 +51,11 @@ gulp.task('create-modules-link', function() {
     }));
 });
 
+gulp.task('electron-rebuild', function(cb) {
+  var electronrebuild = process.platform === 'win32' ? '.\\node_modules\\.bin\\electron-rebuild.cmd' : './node_modules/.bin/electron-rebuild'

```

Comment:
you could just use 
``` JavaScript
let electronrebuild = path.join('node_modules', 'bin', 'electron-rebuild');
```
It will work for windows/macOS/linux. On unix like systems electron-rebuild have no extension. On windows it would run exe/com/cmd file if available.

You can also try to configure it as postinstall tigger as it is explained here https://github.com/atom/node-keytar/issues/51.